src/app/commands/cmd_new_brush.cpp

```diff
@@ -64,6 +64,10 @@ bool NewBrushCommand::onEnabled(Context* context)
 
 void NewBrushCommand::onExecute(Context* context)
 {
+  ASSERT(current_editor);
+  if (!current_editor)
+    return;
+
```

Comment:
1 open aseprite
2 open a .ase tab
3 go back to home tab
4 click new brash
5 crash